src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js

```diff
@@ -83,9 +73,8 @@ function DateOfBirthPage({translate, route, privatePersonalDetails}) {
                     inputID="dob"
                     label={translate('common.date')}
                     defaultValue={privatePersonalDetails.dob || ''}
-                    minDate={minDate}
-                    maxDate={maxDate}
-                    selectedYear={selectedYear}
+                    minDate={moment().subtract(CONST.DATE_BIRTH.MAX_AGE, 'Y').toDate()}
+                    maxDate={moment().subtract(CONST.DATE_BIRTH.MIN_AGE, 'Y').toDate()}
```

Comment:
```suggestion
                    minDate={moment().subtract(CONST.DATE_BIRTH.MAX_AGE, 'years').toDate()}
                    maxDate={moment().subtract(CONST.DATE_BIRTH.MIN_AGE, 'years').toDate()}
```

Maybe `years ` to be more concise with previous usages in other files?